utils/api.js

```diff
@@ -0,0 +1,11 @@
+import  axios  from "axios";
+
+const app = axios.create({
+  baseURL: "https://hot-issue.onrender.com/api",
+});
+
+export const getsAllArticles = () =>{
+    return app.get('/articles').then(({data})=>{
+        return data
+    })
+}
```

Comment:
Nice. Be sure to add a catch block in your use effect so that we can add error handling later :)